powerauth-load-tests/pom.xml

```diff
@@ -83,7 +82,7 @@
                                 <jvmArg>-Xss100M</jvmArg>
                             </jvmArgs>
                             <args>
-                                <arg>-target:jvm-1.8</arg>
+                                <arg>-release:17</arg>
```

Comment:
is the 17 there on purpose? The used Scala version should be compatible with 21.
https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html

Reply:
Indeed. In general, we want to be able to compile with 21 with `target` 17. There is a better option than `target`, see https://www.baeldung.com/maven-java-version#java9